InventoryDAL/ProductTags/ProductTagMySQLDAO.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore;
+using System.Threading.Tasks;
+using InventoryLogic.ProductTags;
+
+namespace InventoryDAL.ProductTags
+{
+    public class ProductTagMySQLDAO : DbContext, IProductTagDAO
+    {
+        private DbSet<ProductTag> ProductTags { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            // use docker composer mysql credentials (safe to store in code)
+            optionsBuilder.UseMySQL("server=db;port=3306;userid=dbuser;password=dbuserpassword;database=accountowner2;");
+        }
```

Comment:
dezelfde connect string moet gebruikt worden, dit is voor de mysql container hetzelfde voor alle entities (2 bij database moet weg).

Reply:
Ah, deze had niet gecommit moeten worden. Dat was een test. Inderdaad, 2 moet weg, maar dan doet hij nog steeds niet wat hij moet doen. Ik zag je oplossing in Teams en zal het proberen netjes te maken.